tests/ode_solve_many_times.cc

```diff
@@ -0,0 +1,33 @@
+#include <iostream>
```

Comment:
good job adding a new test for new functionality. I guess you have verified that the output is correct?

Reply:
The original ODE solver test checked two time points. So the new ODE solver should get the initial condition plus the output from the old ODE solver test if I tell it to use those same two time points, which is what passing the test demonstrates.